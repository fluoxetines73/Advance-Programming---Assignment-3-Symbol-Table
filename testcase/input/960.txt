BEGIN
RPRINT
END
INSERT x string
LOOKUP x
RPRINT
LOOKUP x
BEGIN
END
ASSIGN x '1234ABC'
LOOKUP x
BEGIN
BEGIN
PRINT
ASSIGN x 'abcABC'
ASSIGN aak e 123
PRINT
PRINT
BEGIN
ASSIGN x 'abcABC'
RPRINT
END
END
END