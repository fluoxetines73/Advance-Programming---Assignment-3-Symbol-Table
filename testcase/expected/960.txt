Invalid: ASSIGN aak e 123